### DISCUSSION TITLE
db:seed fails, tests wont run

### DISCUSSION THREAD

**Discussion by BenMcLoughlin** [cypress-io/cypress-realworld-app#1032] [Q&A] [answered]
I'm trying to run the application by running **yarn dev** in one terminal then **yarn cypress:open** in another terminal. The GUI opens but when I click run a test it fails.

 <img width="1003" alt="Screen Shot 2021-07-28 at 8 31 43 AM" src="https://user-images.githubusercontent.com/45271003/127351244-ea3d30de-46f6-42e7-828d-4e5e49ade002.png">

**Comment by kevinold** [comment, marked as answer]
@BenMcLoughlin Please confirm that another service is not running on port 3001.  Do you receive errors from the backend, if you run `yarn start:api`?

**Reply by BenMcLoughlin** [reply]
I ran killall node to ensure there were no other services running. Then I ran yarn start:api and got these errors

TSError: ⨯ Unable to compile TypeScript:
backend/app.ts:79:11 - error TS2769: No overload matches this call.
  The last overload gave the following error.
    Argument of type 'RequestHandler<ParamsDictionary, any, any, ParsedQs, Record<string, any>>' is not assignable to parameter of type 'PathParams'.
      Type 'RequestHandler<ParamsDictionary, any, any, ParsedQs, Record<string, any>>' is missing the following properties from type '(string | RegExp)[]': pop, push, concat, join, and 27 more.

79   app.use(checkAuth0Jwt);
             ~~~~~~~~~~~~~

  node_modules/@types/express-serve-static-core/index.d.ts:61:5
    61     <P extends Params = ParamsDictionary, ResBody = any, ReqBody = any>(path: PathParams, ...handlers: Array<RequestHandlerParams<P, ResBody, ReqBody>>): T;
           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    The last overload is declared here.
backend/app.ts:89:11 - error TS2769: No overload matches this call.
  The last overload gave the following error.
    Argument of type 'RequestHandler<ParamsDictionary, any, any, ParsedQs, Record<string, any>>' is not assignable to parameter of type 'PathParams'.
      Type 'RequestHandler<ParamsDictionary, any, any, ParsedQs, Record<string, any>>' is not assignable to type '(string | RegExp)[]'.

89   app.use(checkCognitoJwt);
             ~~~~~~~~~~~~~~~

  node_modules/@types/express-serve-static-core/index.d.ts:61:5
    61     <P extends Params = ParamsDictionary, ResBody = any, ReqBody = any>(path: PathParams, ...handlers: Array<RequestHandlerParams<P, ResBody, ReqBody>>): T;
           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    The last overload is declared here.
backend/app.ts:94:11 - error TS2769: No overload matches this call.
  The last overload gave the following error.
    Argument of type 'RequestHandler<ParamsDictionary, any, any, ParsedQs, Record<string, any>>' is not assignable to parameter of type 'PathParams'.
      Type 'RequestHandler<ParamsDictionary, any, any, ParsedQs, Record<string, any>>' is not assignable to type '(string | RegExp)[]'.

94   app.use(checkGoogleJwt);
             ~~~~~~~~~~~~~~

  node_modules/@types/express-serve-static-core/index.d.ts:61:5
    61     <P extends Params = ParamsDictionary, ResBody = any, ReqBody = any>(path: PathParams, ...handlers: Array<RequestHandlerParams<P, ResBody, ReqBody>>): T;

**Comment by kevinold** [comment]
@BenMcLoughlin Please ensure you're running the latest `develop` branch as all TypeScript warnings have been resolved on that branch.

**Reply by BenMcLoughlin** [reply]
I deleted my old version and reinstalled it ensuring I was on the latest branch and then fired it up and it worked. Thanks so much! I guess somehow I had one of the depreciated branches. I really appreciate your time. This is one of the most life-changing products I've seen in a while.

**Reply by kevinold** [reply]
@BenMcLoughlin Glad to hear!  Hope you find the RWA examples useful.
